# 3G1 Molecular Engineering[^1]

[^1]:Created by: Tom Xiaoding  Lu on 10/11/18
## L2. DNA RNA Overview
### 2.1 Basic Structure
* Purines: Guanine \& Adenosine are big
* Pyrimidines: Cytosine and Thymine are small
* Weak hydrogen bonds between them, 3 bonds for GC, 2 bonds for AT
<center>

![DeepinScreenshot_select-area_20181015212547](/assets/DeepinScreenshot_select-area_20181015212547.png)

</center>

* Major groove (pink) and minor grooves (blue)
  * Major grooves occur where the backbones (red) are far apart
  * Minor grooves occur where they are close together
  * Easier for DNA binding proteins to interact with the bases on the major grooves

![DeepinScreenshot_select-area_20181015212756](/assets/DeepinScreenshot_select-area_20181015212756.png)

* Two strands of DNA are pointing opposite each other chemically, 5' (front) end and 3' (reverse) end
* All polymerases operate from 5' to 3' direction
### 2.2 Building blocks of DNA and RNA
* Nucleoside triphosphate is the molecule that is polymerised in order to make DNA and RNA
* DNA: sugar is deoxyribose; bases are ATCG
* RNA: sugar is ribose; bases are ACGU
* RNA is relatively unstable due to uracil, the unmethylated form of thymine (it is the first information storage of life)
* DNA is relatively stable due to thymine
* Nucleoside is **sugar + base**

<center>

![DeepinScreenshot_select-area_20181016074745](/assets/DeepinScreenshot_select-area_20181016074745.png)
</center>

### 2.3 RNA Structural Diversity
* A single-stranded RNA molecule can base-pair with itself to form structures that are biochemically important
* Examples:
  * Hammerhead ribozyme: RNA cutting RNA enzyme
  * Transfer RNA a small RNA chain of about 80 nucleotides that transfers a specific amino acid to a growing polypeptide chain at the ribosomal site of protein synthesis during translation
### 2.4 Amino Acid Structure
* Basic end with amino group (N-terminus)
* Carboxyl group with (C-terminus)
* R-group, around 20 different side groups
<center>

![DeepinScreenshot_select-area_20181016081456](/assets/DeepinScreenshot_select-area_20181016081456.png)
</center>

* Lysine and Arginine are basic to tend to bond well with nucleic acid, used by quiet a lot of binding proteins
<center>

![DeepinScreenshot_select-area_20181016081949](/assets/DeepinScreenshot_select-area_20181016081949.png)
</center>

* Valine, Leucine, isoleucine, phenylalanine and methionine all have hydrophobic side chains

![DeepinScreenshot_select-area_20181016082240](/assets/DeepinScreenshot_select-area_20181016082240.png)

* **Serine, Threonine, Cystenine** all have hydroxyl groups, so it can bind with a phosphate ion.
  * Used by kinase, which is an enzyme that catalyzes the transfer of phosphate groups between phosphate-donating molecules to substrates
  * Having a phosphate added or removed switches on or off a protein
  * Serine, threonine kinases
  * Cystenine kinases
* Proline is has side chains that connect back to itself, which typically causes kinks in the chain
* Tryptophan has the largest side group
<center>

![DeepinScreenshot_select-area_20181016152820](/assets/DeepinScreenshot_select-area_20181016152820.png)
</center>

* Cysteine contains sulphur can form cross links between different parts of the protein
<center>

![DeepinScreenshot_select-area_20181016153007](/assets/DeepinScreenshot_select-area_20181016153007.png)
</center>

### 2.5 Peptide Bond
Peptide bonds are used to link two different amino acids through a dehydration reaction:
<center>

![DeepinScreenshot_select-area_20181016153314](/assets/DeepinScreenshot_select-area_20181016153314.png)
</center>
The peptide bond is planar but bond rotation about the alpha carbon provides great flexibility. Computationally impossible to simulate the folding of proteins using molecular dynamics.

<center>

![DeepinScreenshot_select-area_20181016153350](/assets/DeepinScreenshot_select-area_20181016153350.png)
</center>

### 2.6 Protein Structures:
* **Primary structure** refers to the three-dimensional arrangement of atoms in an amino acid-chain molecule
* **Secondary structure** within the long protein chains there are regions in which the chains are organised into alpha=helices and beta-pleated sheets, held together by **hydrogen bonds**
  * Alpha helix:
    * The protein chain is coiled like a loosely-coiled spring. "Alpha" meaning looking down the length of the spring, the coiling is happening in a clockwise direction
    * All the R groups are sticking out sideways from the main helix
    * Each complete turn of the spiral has 3.6 amino acid residues in it

  * Beta pleated sheets:
    * Chains are folded so they lie alongside each other
    * A beta strand is a stretch of polypeptide chain typically 3 to 10 amino acids along with backbone in an extended confirmation
  <center>

  ![DeepinScreenshot_select-area_20181016155511](/assets/DeepinScreenshot_select-area_20181016155511.png)
  </center>

  * **Tertiary** structure of proteins describes the way the whole chain (including secondary structure folds itself into its final 3-D shape)
  * **Quaternary structure** is the number and arrangement of multiple folded protein subunits in a multi-subunit complex
    * Haemoglobin tetramer is an example, with 2 a subunits and 2 b subunits. The binding of a oxygen changes the structure of the protein so adding *non linearity* in the molecule, making it more easily carrying a oxygen molecule given one oxygen already attached.

### 2.7 Lipids
* Have charged heads which like to associate with aqueous solutions and fatty tails that like to avoid water
* It may self-assemble into three structures: Liposome, Micelle and Bilayer sheet (often alpha helical)
* Cells are surrounded by a lipid bilayer membrane, this membrane barrier isolates the cell contents from the environment, similarly sub-cellular structures can be surrounded by lipid membranes.

## L3. Evolution
### 3.1 Three-domain System
* It is a biological classification (*Carl Woese 1977*) that divides cellular life forms into **archaea, bacteria,** and **eukaryote** domains. In particular, it emphasizes the seperation of prokaryotes into two groups.
* **Domain Archea**
  * Prokaryotic (Unicellular organism that lacks a membrane bound nucleus, mitochondria or any other membrane-bound organelle)
  * Have three RNA polymerases like eukaryotes
  * Have cell walls that lack peptidoglycan and have membranes that enclose lipids with hydroocarbons rather than fatty acids
  * Typically extremophiles which have important roles in carbon and nitrogen cycles
  * Matabolism and robustness useful in Biotechnology

* **Domain Bacteria**
  * Prokaryotic
  * Behave less like eukaryotes with only one RNA polymerase
  * Typically a fre $\mu m$ in size
  * Cell wall made of peptidoglycan
  * React to antibiotics in a different way than archea
  * Most of the known pathogenic prokaryotic organisms belong to bacteria
* **Domain Eukarya**
  * Eukaryotes (organisims that contain a membrane bound nucleus)
  * In exhaustive list incudes: Fungi, plants, animals
* Archea/prokaryotes have typically single circular chromosome with extrachromosomal elements called plasmids
  * Plasmids form the basis for cloning vectors used genetic engineering to insert foreign DNA
  * Plasmids are bad for microbial resistance as they typically carry one or to types of microbial resistance
### 3.2 Mitosis
* Interphase:
  * Where the cell spends the longest time doing
  * Chromatin floats freely
* Prophase:
  * DNA replication occurs (actually during S phase)
  * Chromatin condenses into chromosomes
  * Nucleolus disappears
  * Centromeres move to opposite ends of the cell
* Metaphase:
  * Chromosomes line up along the metaphase plate (imaginary plane)
  * Micro-tubules attach to the chromosomes
* Anaphase:
  * Chromosomes break at centromeres and sister chromatids move to opposite ends of the cell

* Telophase and Cytokinesis:
  * Nuclear membrane reforms
  (Nucleoli reappear, chromosomes unwind into chromatin)
  * Myosin II and actin filament ring contract to cleave cell into two (Cytokinesis)
  * Chromosomes form back into chromatin

<center>

![DeepinScreenshot_select-area_20181016165333](/assets/DeepinScreenshot_select-area_20181016165333.png)
</center>

### 3.3 Meiosis
* **Diploid cell**: A cell that contains two sets of chromosomes
* **Haploid cell**: A a cell that contains half the usual number of chromosomes
* The goal of meiosis is to produce from a single diploid cell to four haploids.

Like mitosis, it needs to separate sister chromatids (two havles of duplicated chromosome). Also must separate homologous chromosomes, the similar but nonidentical chromosome pairs an organism receives from its parents. These goals are accomplished in meiosis using a two-step division process, homologue pairs separate during a first round of cell division **meiosis I**, sister chromatids separate during **meiosis II**.
* **Meiosis I**
  * **S-Phase** where all of its chromosomes are replicated
  * **Prophase I:**
    * As in mitosis the chromosomes begin to condense
    * In meiosis I, they also pair up, each chromosome carefully aligns with its homologue partner
    * **Crossing over** occurs helped by *synaptonemal complex* which holds the homologues together
    * This causes the formation of new "remixed" chromosomes with unique combinations of alleles

    ![DeepinScreenshot_select-area_20181016190729](/assets/DeepinScreenshot_select-area_20181016190729.png)

  * **Metaphase**:
    * Spindle begins to capture chromosomes and move them towards the metaphase plate
    * Each chromosome attaches to microtubules from just one pole of the spindle
    * So homologue pairs, not individual chromosomes line up for separation
  * **Annaphase I**:
    * Homologues are pulled apart and move apart to opposite ends of the cell
  * **Telophase I** and Cytokinesis similar as Mitosis forming two haploid cells

* **Meiosis II**
  * There is no S-phase in meiosis II
  * Two haploid cells have just one chromosome from each homologue pair
  * **Prophase II**:
    * Chromosomes condense
    * Nuclear envelopes breakdown (if needed)
  * **Metaphase II**:
    * Chromosomes line up individually along the metaphase plate
  * **Anaphase II** sister chromatids separate and are pulled towards opposite poles
  * **Telophase II & Cytokinesis**

## L3. Central Dogma of Molecular Biology
### 3.1 Summary
* What is the material basis for heredity, the faithful storage & transmission of information from one generation to next?
  * DNA encodes the information to produce proteins in terms of the specific sequence of the amino acids
  * Genes can also encode functional RNAs
* What are the molecules/machinery and operating mechanisms that drive the information flow that generates the observed features of an organism?
  * Two step precess with a RNA intermediate
  * Transcription is DNA -> RNA
  * Translation is RNA -> Protein
  * Information flows from DNA to RNA to protein

### 3.2 Hershey & Chase Experiment 1952
* To address the molecule of heredity
* What people knew is viruses inject the molecule which codes for genetic information into the host cell for it to produce more viruses
* Suspects of this molecules is protein or DNA
* First experiment: all proteins in T2 phage are labelled with radio active sulphur
* The host E.coli is infested by T2 phage
* Observed that most radioactivity is recovered outside of E.coli
* Second experiment DNA of the T2 phage are labelled
* Experiment repeated, now most radioactivity discovered in bacteria
<center>

![DeepinScreenshot_select-area_20181016222053](/assets/DeepinScreenshot_select-area_20181016222053.png)
</center>

### 3.3 Semi-Conservative DNA Replication Model 1953
* There were three models for DNA replication: conservative, semi-conservative and dispersive.
* Meselson and Stahl confirmed that DNA is replicated semi-conservatively through the following experiment:
* Note that the semi-conservative model makes most sense as the two complementary strands of helix might separate during replication, each serving as a template for the construction of a new, matching strand
<center>

![DeepinScreenshot_select-area_20181016222638](/assets/DeepinScreenshot_select-area_20181016222638.png)
</center>

* Experiment begins with growing E.coli in a medium containing only $^{15}\textrm{N}$ until all E.coli's nitrogenous bases were labeled with heavy nitrogen
* Bacteria are then switched to medium containing light $^{14}\textrm{N}$ isotope and allowed to grow for several generations
* Using **density gradient centrifugation** method, after each generation the content of the bacteria samples are measured
* Generation 1:
  * Also produced a single band when centrifuged
  * However this band was higher, intermediate in density between the heavy $^{15}\textrm{N}$  and the light $^{14}\textrm{N}$
  * This means the DNA molecules made in the first round of replication was a hybrid of light and heavy DNA
  * This canfirms that the replication process is either semi-conservative or dispersive
* Generation 2:
  * Produced two bands, one was in the same position as the intermediate band from the first generation
  * Second was higher (appeared to be labelled only with $^{14}\textrm{N}$ )
  * This confirms the semi conservative model since if dispersive, there would never any pure $^{14}\textrm{N}$  band forming
  <center>

  ![DeepinScreenshot_select-area_20181016230835](/assets/DeepinScreenshot_select-area_20181016230835.png)
  </center>

  * The relative proportions of the $^{14}\textrm{N}$ /$^{15}\textrm{N}$ DNAs is defined as a geometric progression over the $i$th generation:
  $$
  D_{14/15}(i)= \left(\dfrac{1}{2}\right)^{i-1} \quad \forall i \geq 1
  $$
  And $^{14}\textrm{N}$/$^{14}\textrm{N}$ proportion simply defined as:
  $$
  D_{14/14}(i)= 1- D_{14/15}  = 1- \left(\dfrac{1}{2}\right)^{i-1} \quad \forall i \geq 1
  $$

### 3.4 Nucleoside VS Nucleotide
 * Nucleo**t**ide contains three things: sugar + base + phosphate
 * Nucleoside contains two things: sugar and base
 * When the 2' carbon is connected to only an H group, the sugar is called deoxiribose. If connected to an HO group, then it is called oxiribose
 * When the 4' carbon is connected to a triphosphate group, it is called nucleoside-triphosphate
 * Monomer subunits of DNA:
    * Deoxyribonucleoside-triphosphate (dNTP)
    * Consists of two purines dATP (deoxy adenosine tri-phosphate) and dGTP (deoxy guanosine tri-phosphate)
    * Two pyrimadines dTTP (deoxy thymidine tri-phosphate) and dCTP (deoxy cytidine tri-phosphate)
 * dATP forms two hydrogen bonds with dTTP
 * dGTP forms three with dCTP
 * **Polymerization of Nucleotides**:
    * The 5' group of a nucleotide triphosphate is held close to the free 3' hydroxyl group of a nucleotide chain
    * The 3' hydroxyl group forms a bond to the phosphorus atom of the free nucleotide. Meanwhile, the bond between the first phosphorus atom and oxygen atom linking it to the next phosphate group breaks
    * A new phosphodiester bond now joins the two nucleotides. A pyrophosphate group has been liberated
    * Pyrophosphate group is hydrolysed, releasing energy and driving the reaction forward to completion
    <center>

    ![DeepinScreenshot_select-area_20181017000308](/assets/DeepinScreenshot_select-area_20181017000308.png)

    ![DeepinScreenshot_select-area_20181017000341](/assets/DeepinScreenshot_select-area_20181017000341.png)
    </center>

### 3.5 DNA Polymerases
  * **They only add on the 3' end**, going from 5' to 3' direction
  * Only when the correct base pair from the nucleoside triphosphate arrives would DNA polymerase catalyse the polymerization
  * The polymerase itself has certain proofreading abilities
  <center>

  ![DeepinScreenshot_select-area_20181017001037](/assets/DeepinScreenshot_select-area_20181017001037.png)
  </center>

### 3.6 DNA Replication

* First, **topoisomerase** unwinds the double strand by releasing the strain energy through temporarily breaking up the phosphate backbone
* **Helicase** is used to break up the hydrogen bonds between the nitrogenous bases
* For the 5' to 3' strand (leading strand):
  * RNA primer is added onto the beginning of the exposed nitrogenous bases by DNA primase
  * DNA polymerase starts to add nucleotides on the 3' end
* For the 3' to 5' strand (lagging strand):
  * RNA primer is added once several tens of exposed bases by the DNA primase
  * DNA polymerase can then add nucleosides onto the 3' end
  * These chunks are called Okazaki fragments
* Once all strands and fragments have been polymerised, the enzyme **exonuclease** removes all RNA primers
* Another DNA polymerase then fills in the gaps, leaving unconnected sugar phosphate backbones (nick site)
* Finally, enzyme ligase joins the sugar-phosphate backbones at each nick site completing replication

<center>

![DeepinScreenshot_select-area_20181017005617](/assets/DeepinScreenshot_select-area_20181017005617.png)
</center>

### 3.7 Transcription: Making of mRNA
* **Gene:**
  * A continuous strand of nucleotides containing a region that codes for an RNA molecule
  * Begins with a promoter (region) and ends with a terminator (region)
  * Also contain regulatory sequences that can be found near the promoter or at a more distant location
* RNA polymerase binds to the promoter DNA
* RNA polymerase creates a **transcription bubble**  which seperates the two strands of the DNA helix, done by breaking the hydrogen bonds between complementary DNA nucleotides
* During **elongation**, the RNA polymerase slides along the DNA strand
* RNA polymerase adds RNA molecules which are complementary to the nucleotides of the 5' to 3' strand
* RNA sugar-phosphate backbone forms with assistance from RNA polymerase to form an RNA strand
* Hydrogen bonds of the RNA-DNA helix break, freeing the newly synthesised RNA strand
* RNA is further processed through **polyaldenylation** (addition of poly(A) tail), **capping** and **splicing** (introns removed, exons joined)
<center>

![DeepinScreenshot_select-area_20181017014537](/assets/DeepinScreenshot_select-area_20181017014537.png)
</center>

### 3.8 Translation: Making of Proteins

#### 3.8.1 Condons and Codon Degeneracy
* A codon is a sequence of three nucleotides which together form a unit of genetic code in a DNA or RNA molecule
* Using 4 bases there are in total $4^3=64$ different combinations of amino acids, in reality however, there are only around 20
* This is due to the degeneracy of codons which is the redundancy of genetic code which prevents point mutation
#### 3.8.2 Process of Translation
* **Initiation:**
  * A ribosome, an mRNA and an initiator tRNA carrying the first amino acid (always methionine) in the protein arrives on the translation site forming an **initiation complex**
  * In Eucaryotes, small ribosomal subunit binds with the mRNA 5' cap upstream of the 5' start codon
  * The small complex then scans to find the start codon, once found the initiator tRNA forms complementary base pairs between its anti codons and the start codons on the mRNA
  * Large ribosomal subunit binds with the initiator tRNA which completes the initiation process forming initiation complex
* **Elongation**
  * There are three sites in the initiation complex, namely P,A,E site
  * The first initiator tRNA occupies the P site, the next tRNA enters the A site which is complementary to the next codon on the mRNA
  * The methionine forms peptide bonds with the next amino acid and the uncharged tRNA moves onto the E site which exits the complex
  * The complex jerks forward and repeats the process until termination
* **Termination**
  * Termination happens when a stop codon in the mRNA (UAA,UAG or UGA) enters the A site
  * Stop codons are recognised by proteins called release factors which fit into the P site
  * Release factors make the enzyme that normally for peptide bonds to add a water molecule to the last amino acid of the chain
  * This reaction separates the chain from the tRNA, a newly made protein is released

## L4. Control of Gene Expression
* **Gene expression**: the process of production of the protein encoded by an individual gene
* **Gene expression**: mechanisms to control whether or not the protein encoded by an individual gene is expressed

**Case study: The Lac Operon**
* Definitions:
  * An **operon** is a functioning unit of DNA containing a cluster of genes under the control of a single promoter
  * The **promoter** is the sequence where the RNA polymerase binds and begins transcription
  * An **operator** is a regulatory sequence that a **repressor** can bind to, when it does the transcription process is halter by physically blocking the binding of the RNA polymerase to the promoter region

<center>

![Screenshot 2018-10-25 at 21.27.41](/assets/Screenshot%202018-10-25%20at%2021.27.41.png)
</center>

In the figure above, we see a sequence of DNA containing the lac operon under the promoter $P_{lac}$ and the gene for repressor $lacI$ (genes are denoted in lowercase). Several cases of the expressions of the lac operon under different environments is described below:
**No lactose:**
* $lacI$ is transcribed into $LacI$ repressor proteins forming a tetramer which binds the operators of the lac operon
* This inhibits the binding between the lac operon promoter and RNA polymerase which halts transcription

<center>

![Screenshot 2018-10-25 at 21.58.49](/assets/Screenshot%202018-10-25%20at%2021.58.49.png)
</center>

**Moderate to high level lactose with no glucose: Positive feedback**
* Lactose enters cell membrane and is broken down by beta-galactosidase into allolactose which is a operon inducer together with glucose and galactose
* Allolactose binds with the $LacI$ repressors causing it to fall off the operators
* This enables RNA polymerase to bind onto the promoter and initiate the transcription of $lacZ$, $lacY$ and $lacA$ mRNAs
* $lacZ$ codes for beta-galactosidase which in turn breaks down lactose to produce allolactose
* $lacY$ codes for permease which binds with the cell membrane to increase the permeability of the cell to glactosides (including lactose, allolactose) via active transport thereby causing a "positive feedback"
* $lacZ$ codes for transacetylase which transfers an acetyl group from acetyl-CoA to glactosides

<center>

![Screenshot 2018-10-25 at 23.10.35](/assets/Screenshot%202018-10-25%20at%2023.10.35.png)
</center>
**Accelerating transcription with cAMP**
* cAMP (cyclic adenosine monophosphate) has a inverse correlation with cell glucose level
* Without the presence of CAP (catabolite activator protein), RNA polymerase alone does not bind well to the lac operon promoter
* cAMP attaches to CAP allowing it to bind onto the CAP site (highlighted in blue before the promoter for the lac operon

<center>

![Screenshot 2018-10-25 at 23.11.29](/assets/Screenshot%202018-10-25%20at%2023.11.29.png)
</center>

## L5. Molecular Cloning
### 5.1 Gene Cloning
* Plasmids are removed from bacterial cells (E. coli) which serve as vectors typically include:
  * Antibiotic resistance genes i.e. ampicillin resistance ($amp^r$), acting as a marker for later selection stage
  * LacZ gene which is responsible for the blue colour, also acting as a marker
  * **Multiple cloning sites** MCS which is the placeholder for the later inserted genes
* DNA containing the gene is also taken from the cell
* A **restriction enzyme** (restriction endonuclease) recognises its specific restriction site and cuts the DNA leaving overhangs called **sticky ends**
* The restriction enzymes also cut open the circular plasmids (using the same enzyme cuts out the gene of interest from its DNA molecule)
* The sticky ends of the restriction fragments attach to each other via base pairing, forming **recombinant plasmids**
* **DNA ligase** makes the bond permanent by attaching nucleotides to each other with **phosphodiester bonds**
* The plasmids are mixed with the bacteria through two main methods (all work like magic):
  * Chemical transformaion
    * wash cells in chilled $CaCl_2$ to permeabilise the membrane
    * 42 degrees heat shock to promote DNA uptake
  * Transformation by electroporation
    * wash cells in water to remove ions
    * 5-20kV/cm shock promotes DNA uptake
  * Both methods are followed by a *recovery period* in rich medium to allow time for expression of the antibiotic resistance gene
* Plasmids with an uninterrupted LacZ gene turn their bacteria blue (unsuccessful ligation)
* Both the recombinant plasmids where the inserted gene interrupts the Lac Z gene and the bacteria which did not take up any plasmids remain uncoloured
* Antibiotics are added, because the plasmid contains the genes for antibiotic resistance, only bacteria which took up the plasmid survive
* The uncoloured bacteria can then be allowed to reproduce

**Limitations:**
* Require very highly purified enzymes which is expensive
* Require restriction sites in the right places which is left to chance

### 5.2 PCR
* Polymerase chain reaction requires:
  * Taq DNA polymerase which is stable under high temperature (95 degrees)
  * DNA primers designed specifically for the DNA region of interest
    * Around 20 nucleotides in length
    * Two primers are used in each PCR reaction designed so they flank the target region
  * PCR buffers to provide the optimal condition for Taq polymerase
  * Nanograms of the DNA strand
* **Denaturation** (96 degrees):
  * Heat the reaction strongly to denature the DNA strands by separating the hydrogen bonds
  * Provides single stranded template for the next step
* **Annealing** (55-65 degrees):
  * Cool the reaction so the primers can bind to their complementary sequences on the single-strand template DNA
* **Extension** (72 degrees):
  * Raise the reaction temperatures so Taq polymerase extends the primers, synthesising new strands of DNA
* Cycle is repeated 25-35 times in a typical PCR reaction which takes 2-4 hours depending on the length of the DNA region

Sketching the process out we see that within each procedure, the expected number of DNA after $T$ cycles assuming perfect annealing and extension processes is, starting from $M$ number of DNA molecules:
$$
M \left(\sum_{n=1}^T 2^n-2 \right)
$$

<center>

![Screenshot 2018-10-26 at 01.57.03](/assets/Screenshot%202018-10-26%20at%2001.57.03.png)
</center>

Things that can go wrong and their solutions:
* No product: check primer design, decrease annealing
 temperature, vary MgCl2 levels
* Multiple products: check primer design, use hot start enzyme,
 increase annealing temperature, vary MgCl2 levels
* Product in negative control: contamination (use laminar air flow
 hood, filter tips)
* Primer dimers: primers self-prime or prime on each other. Redesign primers

### 5.3 Gene Cloning VS PCR
<center>

|   |**Gene cloning** | **PCR**  |
|:-|:-|:-|
| ***Definition***  | Recombinant DNA is constructed invitro <br>and is amplified invivo inside a bacterium  | DNA is amplified in vitro. No need for the <br> construction of rDNA  |
| ***Components***  | Restriction enzymes, DNA ligase, vector <br> DNA and bacterial cells, markers (antibodies)  | Taq DNA polymerase or a thermostable one, <br> RNA primers and free deoxyribonucleotides  |
| ***Quantity required***  | At least a microgram of DNA  | A nanogram of DNA   |
| ***Post synthesis***  | A restriction enzyme is required for <br> reisolation of the amplified DNA from rDNA  | No need for reisolation or use of enzymes  |
| ***Time required***  | 2-4 days  | Up to 4 hours  |
| ***Error possibility***  |High error possibility   | Low error possibility   |
| ***Labour cost***  | Labour intensive   | Fully automated (high set up cost)  |

</center>

### 5.4 Gibson Assembly:
* A DNA ligation technique that can join two blunt ended pieces of DNA
* Uses three enzumes:
  * T5 exonuclease Degrades DNA in 5' to 3' direction at 50 degrees, thereby generating 3' overhangs
  * Taq ligase: Catalyzes the formation of phosphodiester bonds between juxtaposed two adjacent nucleotides
  * Phusion polymerase: facilitates the annealing of the exposed fragments
* Procedures:
  * During PCR, primers are designed to add sticky ends (flaps) at the end of genes
  ![Screenshot 2018-11-20 at 13.23.41](/assets/Screenshot%202018-11-20%20at%2013.23.41.png)
  * 5' to 3' exonuclease chews away the DNA strands in that direction exposing 3' overhangs
  ![Screenshot 2018-11-20 at 13.27.44](/assets/Screenshot%202018-11-20%20at%2013.27.44.png)
  * These complementary strands then anneal, creating the double stranded DNA of interest
  ![Screenshot 2018-11-20 at 13.27.44](/assets/Screenshot%202018-11-20%20at%2013.27.44_tqgfnp2hg.png)
  * DNA polymerase then extends the 3'ends, and Taq ligase seals remaining nicks
  ![Screenshot 2018-11-20 at 13.29.12](/assets/Screenshot%202018-11-20%20at%2013.29.12.png)
  * Result is a fully sealed double stranded DNA suitable for PCR, RCA, etc
  ![Screenshot 2018-11-20 at 13.29.21](/assets/Screenshot%202018-11-20%20at%2013.29.21.png)

**Limitations:**
* Very few, may be expensive for 100,000 bases synthesis
**Advantage:**
* Seamless joining of arbitrary DNA fragments
* Multiple fragments can be joined in one step
* Cheaper than *de novo* synthesis

### 5.5 Large Scale De Novo synthesis
* De novo synthesis refers to the synthesis of complex molecules from simple molecules such as sugars or amino acids
* **Advantages:**
  * Independent of restriction enzymes
  * Independent on ability to PCR fragments of interest
  * Allows complete redesign of DNA sequence
* **Limitations:**
  * High cost
  * Scale: typical ~1kb cloning is now cost-effective for well-funded lab

### 5.6 Seamless joining of DNA Fragments with PCR
* Aim: To join two genes end to end
* Step one: amplifying left one of the fragments like how you would with normal PCR, using one forward one reverse primer
![Capture](/assets/Capture.PNG)
* Step two: Amplify the other fragment with a primer that contains both the forward primer specific for that gene AND the primer used in step one
![Capture2](/assets/Capture2.PNG)
* Step three: Place both products from step one and two in a single PCR reaction, on heating, double strands unwind and the complementary forward and reverse primers from step one bond with each other, Taq polymerase completes the double strand.
![Capture3](/assets/Capture3.PNG)
## L6. Genetic Engineering
### 6.1 Recombinant Protein Expression
* It is the process of expressing non-native proteins in an organism.
* Used to produce a valuable protein product or an organism with favorable properties
* To enable recombinant protein expression, a functional gene is required:
  * **Promoters: initiate transcription**:
    * Can be constitutive or regulated ($P_{lac}$ is IPTG inducible, $P_{BAD}$ is arbinose inducible)
    * Regulation enables control of protein expression
    * Promoters have a range of strengths that determine the levels of protein extracted
  * **Coding sequences**
    * Also known as open reading frames (ORFs)
    * Functional subdomains of proteins can be fused together
    * Affinity tags facilitate the purification of recombinant proteins
    * Short tags direct proteins for specific cellular compartments
  * **Host organisms:** Different hosts have different properties:
    * E. coli is a gram-negative bacterium which is easy to grow, relatively simple cellular phsiology
    * S. cerecisiae is a simple eukaryote which is easy to grow, efficient protein secretion but complex metabolism
    * Mammalian cells are complex multicellular eutaryotes which are more difficult to grow, can produce true human proteins
    * Chlamydomanas reinhardtii are unicellular, eukaryotic algae which are slow growing and photosynthetic
    * After a protein is translated by mRNA, it still undergoes several post-translational modifications through the Golgi complex via transport vesicle
    ![Capture4](/assets/Capture4.PNG)

### 6.2 Genetic Engineering of Insulin
* Insulin has a number of effects that result in decrease blood sugar levels
* The *active* monomer is 51 amino acids
* The *more stable* storage form is a hexamer (a unit with six insulin molecules) with zinc ions interacting with histidine amino acids
* Insulin binds its receptor in target cells resulting in a signaling cascade that includes:
  * Glucose uptake through the glucose transporter 4
  * Storage as glycogen
  * Conversion first to pyruvate
  * Then stored as fat
  ![Capture](/assets/Capture_9gtmbdggk.PNG)
* The structure of insulin:
  * Consists of two separate protein chains linked through covalent disulphide bonds
  * The active monomer is 51 amino acids
  * The more stable storage form is a hexamer with zinc ions interacting with histidine amino acids
* Translation and processing:
  * Translated as preproinsulin
  * Signal peptide directs the protein to the endoplasmic reticulum (ER)
  * The protein folds disulphide bonds form and the signal peptide is removed in the ER
  * After trafficking via the Golgi, the middle fragment (C) is removed leaving amino and carboxy terminal fragments (A&B) bound by disulfide bonds
  * Note C-peptide has been shown to bind to the surface of a number of cells which aids nerve and kidney function
  * The resulting mature protein is 51 amino acids
  <center>

  ![Capture](/assets/Capture_k62ri9mxr.PNG)
  </center>

* Issues with recombinant insulin:
  * Proprotein is processed into mature protein
  * Signal peptide is cleaved hence no initiator methonine left
  * Middle fragment is proteolytically removed
  * Mature protein is A&B polypeptides held together with disulfide bonds
  * Monomer-hexamer transition could be difficult
* Recombinant Insulin expression in E. coli:
  * E.coli lacks the ability to add human like post-translational modifications (PTMs)
  * Therefore make A and B train independently then reconstitute:
    * $lacZ-A$ chain fusion plasmid
    * $lacZ-B$ chain fusion plasmid
    * Transform into E.coli
    * Chemically remove beta galactosidase (lacZ)
    * Purify A and B chains mix and crosslink disulphide bonds
  * Insulin analogues: Lispro (produced through the process above)
    * For some reason, the two amino acids in the B chain results in a decrease in the propensity to form hexamers
    * Since monomers are the active form, Lispro is much faster acting than insulin
    * Much of the commercial recombinant insulin used today is produced in engineered yeast Saccharomyces cerevisiae
### 6.3 Genetic Engineering of Antibody
* Antibodies (Abs) have a range of natural properties:
  * Abs recongnise antigens with exquisite specificity
  * Abs are highly diverse with known structural properties
  * Abs are robust molecules that are naturally long lived
  * Abs are a normal part of the human immune system (which will have consequences with non-human antibodies)
* Structure:
  * Abs are proteins produced by immune cells
  * Bind specific targets
  * Basic structure consists of two heavy and two light chains
  * Most sequence constant, but different antibodies bind different antigens (Ag)
  * $10^{10}$ variants in humans
* Antibody immune response:
  * **Neutralization**: antibodies prevent a virus or toxic protein from binding their target
  * **Opsonization**: a pathogen tagged by antibodies is consumed by a macrophage or a neutrophil
  * **Complement activation:** Antibodies attached to the surface of a pathogen cell activate the complement system
  * The amino acid sequence of **complemnentarity determining regions (CDRs)** aka hypervariable regions (HV) dictate antigen specificity.
* **Generation of B cell/ antibody diversity:**
  - Variability in genes achieved through: VDJ random selection, TdT modification and somatic hypermutation
  * One of the major roles that B cells play in an immune response is the production of antibodies
  * As there are an enormous variety of infectious organisms, the immune system needs to generate a vast numbers of B cells that each produce a different antibody
  * **Shape of variable region** determines the specificity of a particular antibody
  * In the early stages of B-cell development, while still at the **B-cell progenitor state** in the bone marrow:
    * B cells randomly rearrange their **variable (V), diversity (D) and joining (J) genes** (constant region (C) codes the mechanism used to destroy antigen, normally codes for IgM class)
    * Diversity comes from the fact that there are multiple copies of the VDJ genes that can be joined together in different combinations (D+J first, then V+DJ)
    ![Capture](/assets/Capture_6b2ohxd98.PNG)
    * Each antibody molecule is composed of a heavy (containing V & J & D genes) and a light chain (V & J genes)
    * Further diversity is added to the variable region by an enzyme **deoxynucleotidyl transferase (TdT)** that adds extra nucleotides between the V, D and J regions changing the structure of the variable regions that will be produced, this process is called **juction diversity**
      * RAG (recombinant activating gene) cleaves hairpin and adds palindromic nucleotides (reads both ways 5'-3' = 3'-5')
      * TdT adds $N$ nucleotides
      * Mismatches are common, exonucleases remove unpaired nucleotides and the gaps are filled by DNA synthesis and repair machinery.
      * This process adds nucleotides that were not originally encoded in the gene
      ![Capture](/assets/Capture_2xp1kxe05.PNG)
    * B cells can further alter the specificity of the antibody during an infection:
      * A mature B cell meets an antigen that its **B-cell receptor** recongnises
      * That B cell undergoes **somatic hypermutation**
      * Enzyme **activation induced cytidine deaminase (AID)** makes random mutations in the antibody variable region genes
      * If the mutations result in an antibody that more strongly binds to their targets then these B cells will survive and may differentiate into antibody-producing plasma cells
* Antibodies as therapeutics:
  * mAb functions:
      * Monoclonal antibodies (mAb) recognize one epitope with very high specificity
      * Can induce immune responses to pathogens
      * Can interfere with cancer cell function or autoimmune conditions
      * Can conjugate drugs and radioactive atoms to antibodies and deliver them to specific tissues and cells
* **Producing Monoclonal Antibodies**:
  * Challenge mouse with antigen
  * Purify lymphocytes (B cells, mortal, HGPRT+) from spleen
  * Fuse with mylenoma cells (immortal, HGPRT-)
  * Select for hybridoma clones (fused cells, HGPRT+, immortal)
  * Screen for antibody production against target antigen
  ![Capture](/assets/Capture_jfly68604.PNG)

* **Types of synthesised antibodies:**
  * Mouse antibodies are foreign proteins and can bring about an immune response
  * Murine: mouse antibodies (-omab)
  * Human: human antibodies (-mumab)
  * Chimaeric: two thirds human genes (-ximab)
  * Humanised: 90% human, produced through CDR grafting (described below)
* **Humanising antibodies via CDR grafting**:
  * Swap mouse CDR (complementarity determining regions) sequences known to target a particular epitope into a human antibody scaffold
  * Aligning protein sequences allows visualization of the variable and constant regions in antibodies from each organism
  ![Capture](/assets/Capture_v1fyhewf7.PNG)
* **Methods for generating mAbs**
  * I. Humanization: Immunization, hybridoma, cloning then chimerization or humanization via CDR grafting
  * II. Mice carrying Human Ab genes: Engineering the mouse, immunization, hybridoma/B-cell (Ab-gene rescue)
  * III. Phage Display: creating phage library (non-immune/immune/synthetic); selection of Ab from library, described below
* **Phage Display:**
  * Take a few genes that code for specific antibodies and combine them on a strand of DNA
  * Add a phage protein coat made from a virus which only infects bacteria
  * The phage then "displays" the protein on its outside while containing the gene for the protein on its inside (resulting in a connection between genotype and phenotype)
  * A huge collection of these phage displays are made into a phage library
  * Screen Ab libraries for affinity to a target of interest through rounds of selection and propagation
  ![Capture](/assets/Capture_3rwx6it3e.PNG)

## L7. Metabolic Engineering
### 7.1 Definitions:
* **Metabolism**: network of chemical reactions that is responsible for the breakdown of molecules for energy and the synthesis of new molecules for cellular functions
* **Catabolic Bioprocesses**: Break down organic compounds start from complex compounds end up with simpler building blocks, likely to generate energy
* **Anabolic bioprocesses:** Synthesis of cell constituents, start with simple building blocks end up with complex entities, likely to utilize energy
* **Metabolic pathways:** A set of oriented reactions interacting under given physiological conditions via simple intermediates
### 7.2 Metabolic activities:
* Breakdown of complex nutrients to generate building blocks and energy
* Biosynthesis of complex molecules from simple building blocks
* Breakdown / recycling of excess/ damaged complex molecules
* Transport / exchange of chemical compounds across the cellular boundary
* Transport / exchange of chemical compounds across boundaries within the cell

### 7.3 Reaction and Pathways:
* Reactions:
  - $aA+bB\xrightarrow[R]{e}cC+dD$
  - $A$ and $B$ are substrates (reactants)
  - $C$ and $D$ are products
  - $A,B,C,D$ collectively called metabolites
  - $a,b,c,d$ are stoichiometric coefficients of the metabolites
  - $e$ is an enzyme catalyzing reaction $R$
- Pathway:
  - $A\xrightarrow[R1]{e1}B\xrightarrow[R2]{e2}C\xrightarrow[R3]{e3}D \rightarrow F1,F2,F3$
  - $A$ is the substrate, $D$ is the product
  - One or more of the metabolites $F1,F2,F3$ can be produced from $B,D$ and $C$

### 7.4 Enzymes Catalysis mechanisms
* Lock and Key
  ![Capture](/assets/Capture_467xdn4iu.PNG)
* Induced Fit
  ![Capture](/assets/Capture_4hg3agc9p.PNG)
* Michaelis-Menten kinetics:
  - $E+S \xrightleftharpoons[k_{-1}]{k_1} ES \xrightarrow[]{k_2}E+P$
- Enzymatic catalysis lowers the energy of activation of Reactions
  ![Capture](/assets/Capture_a6vtphn2l.PNG)
### 7.5 Widely used metabolic activated carriers
Activated Carrier  | Group carried  
--|--
ATP  | Phosphate  
Acetyl CoA  |  Acetyl group
Carboxylated biotin  |  Carboxyl group
Uridine diphosphate glucose   |  Glucose

Activated carrier = Groupd carried in high energy linkage (HEG) + backbone (BB)
i.e. ATP = Phosphate + ADP

### 7.6 Material Flow:
* Flux: amount of material flowing through a metabolic route (given in $\dfrac{\textit{concentration}}{\textit{time}}$)
* Fulx is a measure of how fast a substrate is converted into a product in a metabolic pathway
### 7.7 Metabolic Pathway Regulation:
* **Metabolic regulation**: concentration of metabolites regulates the flow
* **Heirarchical regulation:** RNA molecules or proteins not directly taking role in metabolic activities control material flow speed
* Different oganisms may have different network topologies and flux distributions:
  - Network topologies - some reaction/pathways may be avaliable in one organism whereas not in others
  - Flux distributions: amount of material flow through a pathway may be different for different organisms

### 7.7 Metabolic Engineering Strategies
* Alter existing (native) genes / pathways to:
  - Remove feedback inhibition
  - Overexpress key enzymes (key = those that impact the flux substantially)
  - Block a metabolic pathway to divert fluxes towards another route
- Introduce foreign genes to:
  - Obtain non-endogenous products (those that are not naturally produced by the host cell)
  - Modify endogenous products so that they acquire a commercially desirable characteristic (less toxic, less smelly, etc)
  - Improve secretory characteristics so that the product is easier to collect and purify
  - Transport/utilize a new substrate(a cheaper or more readily available substrate)
### 7.8 Measuring Metabolite Concentrations
* Definitions:
  - **Metabolome:** complete set of metabolites including low molecular weight indeterminates
  - **Metabolomics:** study of the Metabolome
  - **Fingerprinting:** identification of the endo-metabolome (inside the cell)
  - **Foorprinting:** identification of the exo-metabolome (outside the cell)
- Workflow:
  - GCMS (Gas chromatography/ High performace Liquid chromatography followed by Mass spectrometry/ Nuclear Magnetic Resonance Spectroscopy)
  - Challenges
    - Molecule size varies in only 2 order of magnitude
    - Many orders of magnitude numbers of metabolites
    - Large number of structural isomers
### 7.9 Michaelis-Menten Kinetics:
* $E+S \xrightleftharpoons[k_{-1}]{k_1} ES \xrightarrow[]{k_2}E+P$
* $V_0 = \dfrac{V_{\max}[S]}{K_M+[S]}$
* Where $V_0$ always measures the initial velocity
* [S] measures substrate concentration  
* $K_M=\dfrac{k_{-1}+k_2}{k_1}$, it is the concentration of substrate permitting the enzyme to achieve half $V_{\max}$, an enzyme with high $K_m$ has low affinity for its substrate and requires a greater $[S]$ to achieve $V_{\max}$
* Enzyme kinetics can address many challenges in identifying bottlenecks in flow of material
* Drawback: very difficult to achieve and therefore not manageable at whole metabolism scale

### 7.10 Flux Balance Analysis
* Overview:
  - 1. Curate metabolic Reactions
  - 2. Formulate $S$ matrix
  - 3. Apply mass balance constraints
  - 4. Define objective function $Z$
  - 5. Optimize $Z$ using linear programming

## 8. Genomics
### 8.1 Sanger Chain Termination Sequencing
* Based on the principle that single-stranded DNA molecules that differ in length by just a single nucleotide can be separated by polyacrylamide gel electrophoresis
  ![Capture](/assets/Capture_ausb93ig9.PNG)
* Preparation of identical single-stranded DNA molecules:
  - Can be cloned in a plasmid vector, the resulting DNA will be double stranded so must be denatured into single strands (can be difficult to prepare plasmid DNA that is not contaminated with small quantities of bacterial DNA and RNA)
  - Can be cloned in a bacteriophage M13 vector which is specifically designed for the production of single stranded templates (remove protein coats of the phages through treating with phenol)
- Anneal a short oligonucleotide to the same position of each molecule which subsequently acts as the primer for synthesis of a new DNA strand
  ![Capture](/assets/Capture_cz0bagp3g.PNG)
- The strand synthesis reaction requires DNA polymerase enzyme, abundance of the four deoxyribonucleotide triphosphates (dNTPs-dATP, dCTP, dGTP, dTTP) and a small amount of each of four dideoxynucleotide triphosphates (ddNTPs-ddATP, ddCTPm ddGTP and ddTTP), each labeled with a different fluorescent marker
  - DNA polymerases should have the following properties:
  - High processivity (high length of polynucleotide synthesized before the polymerase terminates through natural causes)
  - Negligible or zero 3' - 5' exonuclease activity (no removal of newly added dideoxynucleotide) which could result in more long sequences than short sequences
- The polymerase does not discriminate between deoxy and dideoxynucleotides, but once incorporated, a dideoxynucleotide blocks further strand elongation as it lacks the 3'-hydroxyl group needed to form a connection with the next nucleotide.
- As normal deoxynucleotides are also present and in larger amount, the result is a set of new molecules all with different lengths each ending in a dideoxynucleotide
  ![Capture](/assets/Capture_qa2bfwej5.PNG)
- To determine the DNA sequence, the strands are sorted according to their lengths using polyacrylamide gel electrophoresis, then molecules are run past a fluorescence detector and a laser capable of discriminating the labels attached to the dideoxynucleotides
  ![Capture](/assets/Capture_wou5ad2lg.PNG)

### 8.2 Genomic Library
* Libraries are made by isolating fragments of interest and **ligating** them into a vector
* Two common types of library:
  - cDNA ("gene" libraries): reverse transcriptase enzyme copies mRNA to DNA
  - Genomic DNA from source fragments that are sunthesised/ restriction fragments/ randomly sheared
- Properties of good libraries:
  - Even representation of source material
  - Few empty vectors
  - DNA not rearranged
  - No vectors with multiple inserts

### 8.3 Heirarchical Shotgun Sequencing
* Construction of a large insert library from nuclear DNA of the species of interest
  - Yeast artificial chromosomes (YACs) and bacterial artificial clone (BAC) are used instead of plasmid as they allow large inserts
  - Create an ordered array of the clone contigs (a set of overlapping DNA segments together represent a region of DNA) through _fingerprinting_ the clones in the library
  - _fingerprinting:_ the clones are cut with a restriction enzyme such as _Hind_ III, the restriction fragment length pattern of each clone represents a fingerprint
  - A series of overlapping clones were then defined for each chromosome. Collectively these overlapping clones define the physical map of the genome
- Each of the large insert clones is fractioned into small DNA fragments through **sonication** (uses high frequency sound waves to make random cuts in DNA molecules)
- The fragments are then modified by adding restriction-site cloning adapters enabling the insertion of fragments into standard plasmid cloning vectors
- The fragments are cloned and subsequently sequenced, the sequence data is then assembled using computer algorithms
- Gaps can appear in the sequence of a BAC or certain regions of the clone might not achieve the full shotgun standard. _Directed sequencing_ is used to sequence these specific regions. Alternatively, primers are developed to sequences bordering the troublesome region
- The molecular map is used to authenticate the finished sequence
### 8.3 Whole Genome Shotgun Sequencing
* Byoasses the generation of a physical map
* DNA itself is sheared and modified by the addition of restriction site adaptors and cloned into plasmids
* These plasmids are then directly sequenced
* Makes use of at least two clone libraries prepared with different types of vector
  - With any cloning vector, it is anticipated that some fragments will not be cloned due to incompatibilities
* Main argument against the use of shotgun sequencing with eukaryotic genomes is the possibility that jumps between repeat units will lead to parts of a repetitive region being left out
  - Overcome by ensuring that one of the clone libraries contains fragments that are longer than the longest repeat sequences in the genome being studied
- Initial result is a series of **scaffolds** with each comprising a set of sequence contigs seperated by sequence gaps that lie between **pair-end reads** - gaps can be closed off by further sequencing that fragment
![Capture](/assets/Capture_l24kwys8i.PNG)
![Capture](/assets/Capture_gscd3s688.PNG)
- Scaffolds themselves are separated by physical gaps which require marker content (fingerprinting) of each scaffold to determine its position on the genome map
  - If a scaffold contains STSs (sequence tagged site) from two noncontiguous parts of the genome then an error has occurred
  - If a pair of end-sequences (100kb or more) do not fall within a single scaffold at their anticipated positions relative to each other then an error has occurred.

### 8.4 Illumina Sequencing
* Reduces cost from \$1000 per million bases to \$0.05-0.15 per million bases with 98% accuracy
* **Tagmentation:**
  - Transposases enzymes randomly cut the DNA into short segments
  - Adapters are added on either side of the cuts through ligation
  - Strands that fail to have adapters ligated are washed away
- **Reduced cycle amplification**
  - Upto 96 different samples (segments) can be run together
  - Sequences for primer binding, indices and terminal sequences are added
  - Indicies are usually six base pairs long and are used to identify samples
  - Terminal sequences are used to attach the strand to the flow cell
  ![Capture](/assets/Capture_xku46tkux.PNG)
  - The flow cell has oligonucleotides that serve to hold the strands in place during sequencing
  ![Capture](/assets/Capture_vqddpwj8u.PNG)
  - Oligos are complementary to the two kinds of terminal sequences
  - As DNA enters the flow cell, one of the adapters attaches to the complementary oligo
  ![Capture](/assets/Capture_p0dza4tea.PNG)
- **Bridge amplification**
  - Goal is to create hundreds of identical strands (some forward, some reverse) of DNA through the following cycles:
  - Polymerases move along a strand of DNA creating its complementary strand
  ![Capture](/assets/Capture_sxdu7ssyo.PNG)
  - Original strand is washed away leaving the reverse strand, at the top of the reverse strand there is an adapter sequence
  - DNA strand bends and attaches to the oligo that is complementary to the top adapter sequence
  ![Capture](/assets/Capture_dpuy8ruy3.PNG)
  - Polymerases attach to the reverse strand and synthesizes the original strand
  - The double stranded DNA is denatured so each strand can be separately attached to an oligonucleotide sequence anchored to the flow cell
  ![Capture](/assets/Capture_3uvcwov8j.PNG)
  - The resulting cycles is all the DNA strands in one area are from a single source
  ![Capture](/assets/Capture_xxy9r7eaa.PNG)
- **Sequence by synthesis**
  - All of the reverse strands are washed off leaving only forward strands
  - Primers attach to the forward strands and a polymerase adds fluorescently tagged nucleotides to the DNA strand, this can be read by a camera
  ![Capture](/assets/Capture_m7u1sf886.PNG)
  ![Capture](/assets/Capture_tfv89m2lv.PNG)
  - The indices are read in a similar manner (wash the newly synthesized strand off, attach index 1 primer and polymerizes the index 1 sequence with fluorescently tagged nucleotides)
  ![Capture](/assets/Capture_4tfxdmc8n.PNG)
  - The strand forms a bridge again, reverse strands synthesized, original strands washed off
  - Process repeated to read the reverse strand and tag 2
  ![Capture](/assets/Capture_ktf3up18g.PNG)
- **Data analysis**
  - Sequencing occurs for millions of clusters at once each cluster has 1000 identical copies of a DNA insert
  - Sequence data analyzed by finding fragments with overlapping areas (contigs)
  ![Capture](/assets/Capture_jgabeaa71.PNG)
  - Contigs are compared to molecular map for identification
